libcnb-test/tests/integration_test.rs

```diff
@@ -22,7 +22,10 @@ ERROR: failed to build: failed to fetch builder image 'index.docker.io/libcnb/vo
 ")]
 fn panic_on_unsuccessful_pack_run() {
```

Comment:
Are there any more cases where it would be useful to have integration tests here? Either related to this PR (in which case potentially in scope for this PR), or for existing libcnb-test features (in which case it would be good to file a GitHub issue to track adding them later).

Reply:
This is an interesting problem. So far, I added the "tests" for `libcnb-test` to the examples since we would need custom buildpacks to test specific features of `libcnb-test` and I have no idea where to put them. For example, the Ruby example now runs a subsequent test.

Problem with this approach that it isn't really clear we rely on the examples to test critical functionality.